Stack/Opc.Ua.Core/Schema/NodeIds.csv

```diff
@@ -14387,3 +14379,129 @@ AuthorizationServicesConfigurationFolderType_ServiceName_Placeholder,23557,Objec
 AuthorizationServicesConfigurationFolderType_ServiceName_Placeholder_ServiceUri,23558,Variable
 AuthorizationServicesConfigurationFolderType_ServiceName_Placeholder_ServiceCertificate,23559,Variable
 AuthorizationServicesConfigurationFolderType_ServiceName_Placeholder_IssuerEndpointUrl,23560,Variable
+ProgramDiagnostic2DataType,24033,DataType
```

Comment:
is the change of nodeid for this type really intentional?

Reply:
From Errata 1.04.10

ProgramDiagnostic2DataType is still using the broken StatusResult   for LastMethodReturnStatus and not StatusCode like defined in the   specification.

Change the   ProgramDiagnostic2DataType.LastMethodReturnStatus DataType to StatusCode.  Change the NodeId for the DataType Node   and its associated Encoding from 15396...15405 to 24033...24042.